site/content/en/docs/Getting Started/create-webhook-fleetautoscaler.md

```diff
@@ -429,6 +429,27 @@ kubectl delete -f https://raw.githubusercontent.com/GoogleCloudPlatform/agones/m
 
 Note that now secure communication is established and we can trust our webhook. If we need to use server outside of the kubernetes cluster we can use other Root certificate authority and put it into as caBundle parameter in fleetautoscaler configuration (in pem format, base64-encoded).
 
+## Troubleshooting Guide
+
+There could be some problems with configuration of fleetautoscaler and webhook service.
+Easiest way to debug this is to run:
+```
+kubectl describe fleetautoscaler <FleetAutoScalerName>
+```
+Then you would see events at the bottom of the output.
+
+### Common error messages.
+
+Error when you configure wrong Service Path for the FleetAutoscaler:
+```
+Error calculating desired fleet size on FleetAutoscaler simple-fleet-r7fdv-autoscaler. Error: Bad status code from the server: https://autoscaler-tls-service.default.svc:8000/scale. Status: 404
```

Comment:
is "Status: 404" part of err.Error()?

Reply:
Moved this into the middle of a sentence